core/two-steps.ts

```diff
@@ -0,0 +1,45 @@
+import { router } from '@slowreader/core'
+import { atom } from 'nanostores'
+
+interface BackRoutes {
+  [key: string]: string
+}
+
+export type Side = 'first' | 'second'
+
+export const backRoutes: BackRoutes = {
+  categories: '/feeds/categories',
+  fast: '/fast/:category',
+  slow: '/slow/:feed'
+}
+
+export const side = atom<Side>('first')
```

Comment:
Maybe we can use boolean since we do not plan third page?

It could simplify code and types.

We can rename store to `secondStep`